package.json

```diff
@@ -88,5 +88,34 @@
   "dependencies": {
     "@types/lodash.clonedeep": "^4.5.6",
     "lodash.clonedeep": "^4.5.0"
+  },
+  "exports": {
+    ".": "./index.js",
+    "./package.json": "./package.json",
+    "./actions/adjust": "./actions/adjust.js",
+    "./actions/customFunction": "./actions/customFunction.js",
+    "./actions/delivery": "./actions/delivery.js",
+    "./actions/effect": "./actions/effect.js",
+    "./actions/namedTransformation": "./actions/namedTransformation.js",
+    "./actions/psdTools": "./actions/psdTools.js",
+    "./actions/reshape": "./actions/reshape.js",
+    "./actions/resize": "./actions/resize.js",
+    "./actions/rotate": "./actions/rotate.js",
+    "./actions/roundCorners": "./actions/roundCorners.js",
+    "./actions/transcode": "./actions/transcode.js",
+    "./actions/variable": "./actions/variable.js",
+    "./actions/videoEdit": "./actions/videoEdit.js",
+    "./qualifiers/animatedFormat": "./qualifiers/animatedFormat.js",
+    "./qualifiers/aspectRatio": "./qualifiers/aspectRatio.js",
+    "./qualifiers/background": "./qualifiers/background.js",
```

Comment:
It looks like we can export entire folders instead of manually listing these one by one:

https://nodejs.org/api/packages.html

```
{
  "name": "my-mod",
  "exports": {
    ".": "./lib/index.js",
    "./lib": "./lib/index.js",
    "./lib/*": "./lib/*.js",
    "./feature": "./feature/index.js",
    "./feature/*": "./feature/*.js",
    "./package.json": "./package.json"
  }
}
```

Reply:
done